### PR TITLE
don't allow empty/negative form fields

### PR DESCRIPTION
**Pull request by nkompella**
Trivial change that requires users to fill out all fields except for cuisine, as well as restricting negative inputs to the form.

### CODE REVIEW

File: app/views/users/edit.html.haml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 = form_tag edit_path, :class => '' do
 
   = label_tag(:age, 'Age')
-  = number_field_tag :age, 'age', class: ""
+  = number_field_tag :age, 'age', min: 1, required: true
```
Comment (techprojsforfun) [inline app/views/users/edit.html.haml]: perhaps we should keep the class attribute for later CSS/bootstrap styling on the form

File: app/views/users/edit.html.haml [hunk 2]
```diff
@@ -12,10 +12,10 @@
   = label_tag :gender, 'Female'
 
   = label_tag(:weight, 'Weight')
-  = number_field_tag(:weight, 'weight')
+  = number_field_tag(:weight, 'weight', min: 1, required: true)
 
   = label_tag(:height, 'Height')
-  = number_field_tag(:height, 'height')
+  = number_field_tag(:height, 'height', min: 1, required: true)
```
Comment (techprojsforfun) [inline app/views/users/edit.html.haml:18]: we can add a Cuke/Capybara scenario to test the sad path if the user does not enter their weight/height and it should reject the input!

File: app/views/users/edit.html.haml (hunk 1 above)
Comment (nkompella) [reply app/views/users/edit.html.haml]: resolved, I added it back in

File: app/views/users/edit.html.haml (hunk 2 above)
Comment (nkompella) [reply app/views/users/edit.html.haml:18]: We're in the process of adding tests for the entire form (tech debt from iteration 1), merging this in will make it easier for whoever writes tests to write comprehensive tests for the entire form functionality.

**Pull request merged**